queue_job/fields.py

```diff
@@ -23,7 +24,7 @@ class JobSerialized(fields.Field):
     """
 
     type = "job_serialized"
-    column_type = ("text", "text")
+    _column_type = ("jsonb", "jsonb")
```

Comment:
to preserve existing jobs this should be migrated, no?

Reply:
I will add it after fixing the pre-commit bug